### DISCUSSION TITLE
Right click - Disable node

### DISCUSSION THREAD

**Discussion by shak2** [godotengine/godot-proposals#9962] [Editor]
Disable node, not just visibility with the eye, i'd like to actually disable a node fully but not delete it. Could have a strikethrough on the text to show it is disabled (recommended), or generic grey out the text.

Right click - disable node.

Right click - enable node.

When useful: Useful when prototyping multiple ways of doing the same thing, like camera movement scripts which require different camera node structures. Turning visibility off here does nothing so currently just have to manually reorder them everytime or detach, reattach scripts. Painful.

**Comment by AThousandShips** [comment]
Thank you for your proposal, consolidating in:
* https://github.com/godotengine/godot-proposals/issues/7715
